.github/workflows/tests.yml

```diff
@@ -13,22 +13,32 @@ jobs:
             goarch: amd64
             psql: true
             skip_wrapper_tests: false
+            sonic: false
           - os: ubuntu-latest
             goarch: 386
             psql: true
             skip_wrapper_tests: true
+            sonic: false
           - os: macos-latest
             goarch: amd64
             psql: true
             skip_wrapper_tests: true
+            sonic: false
           - os: macos-13 # beta
             goarch: amd64
             psql: true
             skip_wrapper_tests: true
+            sonic: false
           - os: windows-latest
             goarch: amd64
             psql: true
             skip_wrapper_tests: true
+            sonic: false
+          - os: ubuntu-latest
```

Comment:
I'm not seeing which one is sonic in anyway from the build list.
<img width="736" alt="image" src="https://github.com/user-attachments/assets/74cc3708-adaa-47b0-a5c2-a6d913e5e2a6" />
Or is there two that use sonic (the last two ubuntu builds)? Am I blind? 🙈

Reply:
The last one is a docker build so it only comes up with false true, but the matrix build above, seems to be the only one with sonic set to true, but in the logs, it doesn't seem to pull the bytedance package down. I am trying to fix that now.

Reply:
![image](https://github.com/user-attachments/assets/26b47b5e-ed65-4285-a7d2-2b9fdb0c05b3)
Unintended catch by you sir 🚀 but that should be good to go now.

Reply:
Looks like the docker build is ded, but otherwise, looks good

Reply:
Do you want me to fix the docker build in this PR? @gloriousCode

Reply:
Sorry for blaming you for it.